INDI/INDI/indiserver.c

```diff
@@ -156,16 +155,21 @@ typedef struct
     int pid;            /* process id or REMOTEDVR if remote */
     int rfd;            /* read pipe fd */
     int wfd;            /* write pipe fd */
-    int efd;            /* stderr from driver, if local */
     int restarts;       /* times process has been restarted */
     LilXML *lp;         /* XML parsing context */
     FQ *msgq;           /* Msg queue */
     unsigned int nsent; /* bytes of current Msg sent so far */
-} DvrInfo;
+    struct strDvrInfo* pNextToRestart; /* next to restart, or NULL */
```

Comment:
DrvInfo structures that are trying to restart at 0.1Hz are put into a linked list;

pNextToRestart is the pointer (link) to the next DrvInfo that is trying to start, or it is NULL indicating the end of the list.